eng/Version.Details.xml

```diff
@@ -270,25 +270,21 @@
       <Uri>https://github.com/dotnet/runtime-assets</Uri>
       <Sha>3453104731a7fd5af1b7613bb9dc9dfa6263b312</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Net.Compilers.Toolset" Version="4.4.0-2.22463.8">
+    <Dependency Name="Microsoft.Net.Compilers.Toolset" Version="4.4.0-2.22464.20">
       <Uri>https://github.com/dotnet/roslyn</Uri>
-      <Sha>76b6eb3e6b207a3bfe872f17b17008d2165da5f3</Sha>
+      <Sha>91902d4dd2c07549fb64357ed5c3a0ee9d7c3610</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.CodeAnalysis" Version="4.4.0-2.22463.8">
+    <Dependency Name="Microsoft.CodeAnalysis" Version="4.4.0-2.22464.20">
```

Comment:
IIRC from my last conversation with @sharwell, bumping `Microsoft.CodeAnalysis` results in a newer VS version being required.

Reply:
It depends where we use this reference. I need to check which package specifically we reference with the EventSourceGenerator as that’s the only generator that might reference this package and that we care about VS experience (though since it doesn’t affect the design time experience, having it silently fail in VS but still work in the build isn’t that bad). IIRC, we don’t care about the VS experience for the src/tests test projects, and the generator there doesn’t affect the design time experience anyway.

Reply:
Presumably a dev could add a reference to this dependency elsewhere in this repo and suddenly (without knowing) locks out devs from contributing that don't have access to the IntPreview VS builds. A good example would be in System.Private.CoreLib which ascendingly is referenced by many libraries and is part of many solution files under src/libraries. If today Microsoft.CodeAnalysis is only used in a component that wouldn't impact external contributors then the dependency is probably OK to have. At the least we need comments in Versions.props and maybe also in a markdown file explaining this dependency and what to watch out for.